pkg/setting/setting.go

```diff
@@ -802,19 +803,18 @@ func (cfg *Cfg) loadSpecifiedConfigFile(configFile string, masterFile *ini.File)
 	return nil
 }
 
-func (cfg *Cfg) loadConfiguration(args CommandLineArgs) (*ini.File, error) {
+func (cfg *Cfg) loadConfiguration(args CommandLineArgs) (parsedFile *ini.File, err error) {
 	// load config defaults
 	defaultConfigFile := path.Join(HomePath, "conf/defaults.ini")
 	configFiles = append(configFiles, defaultConfigFile)
 
-	// check if config file exists
-	if _, err := os.Stat(defaultConfigFile); os.IsNotExist(err) {
-		fmt.Println("Grafana-server Init Failed: Could not find config defaults, make sure homepath command line parameter is set or working directory is homepath")
```

Comment:
> Users may remove it or override the defaults there. 

Would it make sense to provide some direction in the output? I don't have exact wording to propose right now but basically not removing this line and instead adapting it to the current setup. 

This would serve two purposes: 
(1) make sure that users are aware that the config which is currently being used is the default one and not the one they thought they had set (when overriding the defaults for example)
(2) indicate where/how to set the configuration